pp-installer.php

```diff
@@ -19,15 +19,7 @@
 }
 
 add_action('admin_head-widgets.php', 'p7i_prevent_delete_inactive_widgets');
-
-if (p7i_is_active_theme()) {
-    return;
-}
-
-add_action('plugins_loaded', 'p7i_test_drive_init');
 add_action('wp_ajax_ppi_api', 'p7i_api_route_request');
-add_action('admin_menu', 'p7i_add_menu_item');
-add_action('load-toplevel_page_p7-installer', 'p7i_admin_page_init');
-add_action('admin_enqueue_scripts', 'p7i_pointer_init');
+add_action('plugins_loaded', 'p7i_init');
```

Comment:
The logic here subtly changed, and I'm wondering if we've thought it all the way through.  Before the change you made, none of these actions would be added if `p7i_is_active_theme()` returned false.  Now, we have several actions hooked no matter what, which is different than before, since the guard only applies within the new function.

Reply:
Tested this again in a non-mu environment and everything seemed to work fine.  ¯\_(ツ)_/¯

The changes I made are only needed for our design store setup, so theoretically we could not merge into master and just use a zip of this branch for the design store until we don’t need to run 6 anymore....  ¯\_(ツ)_/¯¯\_(ツ)_/¯¯\_(ツ)_/¯